`panic: runtime error: index out of range [1] with length 1`

**Describe the issue:**

I [downloaded Camunda 8 Run, extracted, tried to run it](https://docs.camunda.io/docs/self-managed/setup/deploy/local/c8run/ ) as instructed.

**Actual behavior:**

```
> .\c8run.exe
panic: runtime error: index out of range [1] with length 1

goroutine 1 [running]:
main.main()
        main/c8run_windows.go:160 +0x1a39
>
```

**Expected behavior:**

I would expect more useful output. Is it missing a command line argument? If so, what is the expected value?

**How to reproduce:**

Run `.\c8run.exe` on Windows.

**Logs:**

See above, that's all I got.

**Environment:**

- Platform: Windows
- Architecture: x86
- Version: 8.6.6

--- a/c8run/main.go
+++ b/c8run/main.go
@@ -249,7 +249,7 @@ func main() {
 	} else if os.Args[1] == "-h" || os.Args[1] == "--help" {
 		usage(0)
 	} else {
-		panic(fmt.Sprintln("Unsupported operation", os.Args[0]))
+		panic(fmt.Sprintln("Unsupported operation", os.Args[1]))
 	}
 	fmt.Print("Command: " + baseCommand + "\n")
 

--- a/c8run/main.go
+++ b/c8run/main.go
@@ -191,6 +191,11 @@ func stopDocker(extractedComposePath string) error {
 	return nil
 }
 
+func usage(exitcode int) {
+	fmt.Printf("Usage: %s [command] [options]\nCommands:\n  start\n  stop\n  package\n", os.Args[0])
+	os.Exit(exitcode)
+}
+
 func main() {
 	c8 := getC8RunPlatform()
 	baseDir, _ := os.Getwd()
@@ -231,7 +236,9 @@ func main() {
 	baseCommand := ""
 	// insideConfigFlag := false
 
-	if os.Args[1] == "start" {
+        if len(os.Args) == 1 {
+                usage(1)
+        } else if os.Args[1] == "start" {
 		baseCommand = "start"
 	} else if os.Args[1] == "stop" {
 		baseCommand = "stop"
@@ -240,10 +247,9 @@ func main() {
 	} else if os.Args[1] == "clean" {
 		baseCommand = "clean"
 	} else if os.Args[1] == "-h" || os.Args[1] == "--help" {
-		fmt.Println("Usage: c8run [command] [options]\nCommands:\n  start\n  stop\n  package")
-		os.Exit(0)
+		usage(0)
 	} else {
-		panic("Unsupported operation")
+		panic(fmt.Sprintln("Unsupported operation", os.Args[0]))
 	}
 	fmt.Print("Command: " + baseCommand + "\n")
 

